romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -271,10 +273,8 @@ def create_image_model(input_model, image_info):
 
     # Create output datamodel
     # ... and add all keys from input
-    meta = {}
-    meta.update(input_model.meta)
+    meta = input_model.meta.model_dump()
     meta["cal_step"]["ramp_fit"] = "INCOMPLETE"
-    meta["photometry"] = maker_utils.mk_photometry()
```

Comment:
I think I infer that WfiImageModel.make_default(...) is now doing this for us?

Reply:
Correct.